common/system.h

```diff
@@ -127,6 +127,28 @@ enum Type {
 
 } // End of namespace LogMessageType
 
+namespace Common {
```

Comment:
No, please move it to `common/textconsole.h`

Reply:
I couldn't because it depends on `LogMessageType::Type` which is declared in `system.h`.

Reply:
I created a `common/log.h`file and move I move `LogMessageType::Type`and `setLogWatcher` method in it.